packages/@react-spectrum/color/docs/ColorSlider.mdx

```diff
@@ -0,0 +1,228 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import colorTypes from 'docs:@react-types/color/src/index.d.ts';
+import docs from 'docs:@react-spectrum/color';
+import {HeaderInfo, PropTable, TypeLink} from '@react-spectrum/docs';
+import {Layout} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/color/package.json';
+import statelyDocs from 'docs:@react-stately/color';
+
+export default Layout;
+
+```jsx import
+import {ColorSlider} from '@react-spectrum/color';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Color
+keywords: [color slider]
+---
+
+# ColorSlider
+
+<p>{docs.exports.ColorSlider.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorSlider']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/color-slider/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ColorSlider defaultValue="#7f0000" channel="red" />
+```
+
+## Value
+
+A ColorSlider requires either an uncontrolled default value or a controlled value, provided using the `defaultValue` or `value` props respectively.
+The value provided to either of these props should be a string or <TypeLink links={colorTypes.links} type={colorTypes.exports.Color} /> object that matches the provided ColorSlider's `channel` prop. For example, a
+ColorSlider representing a color channel ("red", "green", "blue") could have a hex color code (e.g. "#7f0000") or a decimal code (e.g. "rgb(127, 0, 0)") provided.
+For a full list of supported channels, see the [Props](#props) table below.
```

Comment:
I don't quite understand the second half of this paragraph.

```suggestion
The value provided to either of these props should be a color string or <TypeLink links={colorTypes.links} type={colorTypes.exports.Color} /> object. A `channel` prop must also be provided to specify which color channel the slider should display.
```

Reply:
I was trying to explain that `channel` and the `value` provided have to be compatible. For instance you can't pass in a HSL value and use the "red" color channel.

Reply:
Ah ok, updated my suggestion.